pyconbalkan/urls.py

```diff
@@ -44,6 +44,7 @@
     path('robots.txt', TemplateView.as_view(template_name='robots.txt', content_type='text/plain')),
     path('organizers/<slug:slug>/', organizer_view, name='organizer_detail'),
     path('organizers', organizers_listview, name='organizers'),
+    path('volunteer/create/', volunteer_createview, name='volunteer_create'),
```

Comment:
@maroshmka let's use `volunteers` (in plural) to keep the consistence with organizers url. :)

Reply:
done